src/modal/modal.component.ts

```diff
@@ -180,14 +185,20 @@ export class Modal implements AfterViewInit, OnChanges {
 		}
 	}
 
-
 	/**
 	 * Set document focus to be on the modal component after it is initialized.
 	 */
 	ngAfterViewInit() {
+		const modalContent = this.modal.nativeElement.querySelector(".bx--modal-content");
```

Comment:
modal content should be a directive (`ibmModalContent`), and `hasForm` should be an input on it 👍 That way we're not messing around with classnames directly (kinda fragile)